eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/ChannelTlv.scala

```diff
@@ -84,6 +88,7 @@ object OpenDualFundedChannelTlv {
   val openTlvCodec: Codec[TlvStream[OpenDualFundedChannelTlv]] = tlvStream(discriminated[OpenDualFundedChannelTlv].by(varint)
     .typecase(UInt64(0), upfrontShutdownScriptCodec)
     .typecase(UInt64(1), channelTypeCodec)
+    .typecase(UInt64(0x40000001), requireConfirmedInputsCodec)
```

Comment:
What's the rationale behind this choice of tag?

Reply:
Completely random, hopefully we agree with CLN on a standard value for that TLV in the next couple of weeks and can fix this before the next eclair release!